documents/postman/staging-simplified.postman_environment.json

```diff
@@ -0,0 +1,71 @@
+{
+	"id": "1acffa98-51c5-4ba7-87fc-9f3d460c38ba",
+	"name": "development Copy",
```

Comment:
development Copy shouldn't be the env name that shows in Postman. How about staging-vanotify? This way it is client-friendly but also won't overwrite our staging env for the full api collection.